### PR TITLE
ENH: Changes the content review email template from a textareafield to an HTMLEditorField

### PR DESCRIPTION
**Pull request by Radroaches**
Currently users have to enter their own HTML tags into the editor if they want to style the review email. 

This would also allow users to preview the HTML before sending the email.

### CODE REVIEW

File: src/Extensions/ContentReviewDefaultSettings.php [hunk 1]
```diff
@@ -164,13 +165,51 @@ public function updateCMSFields(FieldList $fields)
...
+            'formatselect',
+            $editorButtonsGroupSeparator,
+            'code',
+        ];
+
+        $config->setButtonsForLine(1, $allowedEditorButtons);
+        $config->setButtonsForLine(2, []);
+        $config->setButtonsForLine(3, []);
+
+        return $config;
```
Comment (ChloeSartorelli) [inline src/Extensions/ContentReviewDefaultSettings.php]: The config should be defined in `_config.php` and be given its own name - this allows developers to easily modify it in their own projects however they may so choose. Also there's no need to set a variable for `|`.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
My main concern with this is that somebody may have some html set in this field which will now be stripped out by the tinymce config, which makes this a breaking change.

File: src/Extensions/ContentReviewDefaultSettings.php (hunk 1 above)
Comment (jareddreyerss) [reply src/Extensions/ContentReviewDefaultSettings.php]: @GuySartorelli removed the tinymce helper as we can put this in our projects as agreed doesn't need to live in this module.

**Comment by GuySartorelli** [comment]
Sorry it's taken me so long to come back to this one.
Is there an easy way to define a specific config for that one `HTMLEditorField` instance? Or do we need a `private static string $content_review_body_config_name = '';` or similar, together with something like this?
```php
$configName = $this->owner->config()->get('content_review_body_config_name');
if ($configName) {
    $wysiwyg->setEditorConfig(HTMLEditorConfig::get($configName));
}
```

**Comment by jareddreyerss** [comment]
> Sorry it's taken me so long to come back to this one. Is there an easy way to define a specific config for that one `HTMLEditorField` instance? Or do we need a `private static string $content_review_body_config_name = '';` or similar, together with something like this?
> 
> ```
> $configName = $this->owner->config()->get('content_review_body_config_name');
> if ($configName) {
>     $wysiwyg->setEditorConfig(HTMLEditorConfig::get($configName));
> }
> ```

😄  just thinking now... we did not actually need a PR for this change at all 🥴 . 

What would you guys like? am happy to put in a config member as above so it's easily switchable and doesn't break current projects.  Otherwise I am also happy to close this PR since its a small change and can be done by the developer via project code.

**Comment by GuySartorelli** [comment]
Nobody else seems to be clamoring for this to be merged - I'm not fussed either way tbh :p
If you're happy to resolve this in projects and close the PR then that's absolutely fine. If someone else wants this functionality in the future they're welcome to resurrect this PR.

**Comment by GuySartorelli** [comment]
Closing as there was no response since my last comment.

**Pull request closed without merging**